wolfSSL/src/chapter02.md

```diff
@@ -628,6 +1002,86 @@ Enables Blake2s algorithm support
 
 Enables Signaling Cipher Suite Value(SCSV) support on the server side. This handles the cipher suite 0x56 0x00 sent from a client to signal that no downgrade of TLS version should be allowed.
 
+#### HAVE_AEAD
+
+Implements the use of AEAD and is required for TLS 1.3.
+
+#### HAVE_AES_CBC
+
+Enable option for AES.
+
+#### HAVE_AES_DECRYPT
+
+This setting controls whether the decrypt direction of the various AES modes is supported.
```

Comment:
Please use `NO_AES_DECRYPT` for documentation. The settings.h turns this on internally by default. For reducing code size `NO_AES_DECRYPT` can be set to disable the decrypt and only support encryption.